app/backend/src/couchers/servicers/admin.py

```diff
@@ -288,6 +291,34 @@ def CreateCommunity(self, request, context, session):
 
         return community_to_pb(session, node, context)
 
+    def UpdateCommunity(self, request, context, session):
+        cluster = session.execute(select(Cluster).where(Cluster.id == request.community_id)).scalar_one_or_none()
+        if not cluster:
+            context.abort(grpc.StatusCode.NOT_FOUND, errors.COMMUNITY_NOT_FOUND)
+
+        node = session.execute(select(Node).where(Node.id == cluster.parent_node_id)).scalar_one_or_none()
+        if not node:
+            context.abort(grpc.StatusCode.NOT_FOUND, errors.COMMUNITY_NOT_FOUND)
+
+        if request.name:
+            cluster.name = request.name
+
+        if request.description:
+            cluster.description = request.description
+
+        if request.geojson:
+            geom = shape(json.loads(request.geojson))
+            if geom.geom_type != "MultiPolygon":
+                context.abort(grpc.StatusCode.INVALID_ARGUMENT, errors.NO_MULTIPOLYGON)
+            node.geom = from_shape(geom)
```

Comment:
Could you please refactor this into a function, something like `load_community_geom(geojson, context)`, then use it as well in `CreateCommunity`?